CONTRIBUTING.md

```diff
@@ -86,3 +86,41 @@ described below:
   "proposal finalized" label.  This indicates that it is ready to be
   implemented by a platform developer, see the [process](#contributing) above.
 - Once a Pull Request is approved, it will be merged into the 'master' branch.
+
+## Spec Release Process
+
+Any member of the OSBAPI PMC can request that the accepted spec changes on
+master are released in a new version of the spec. They will do this by
+creating a new PR with the tile of the proposed release. For example,
+"Release Proposal: v2.20".
+
+### Prepare a PR
+
+- In a fork create a new branch called "v$major.$minor".
+- Update [README](README.md)'s "Latest Release" section and link to branch URL.
+- Update [RELEASE-NOTES](release-notes.md) with details of the changes included
```

Comment:
Meta-thing: is it easier to do this step in each PR that alters the spec in master, rather than doing it at the end of the cycle?

We should consider using github milestones to track issues/PRs, too.